packages/editor/src/lib/editor/tools/StateNode.ts

```diff
@@ -10,6 +12,17 @@ import {
 } from '../types/event-types'
 
 type TLStateNodeType = 'branch' | 'leaf' | 'root'
+const STATE_NODES_TO_MEASURE = [
```

Comment:
feels like we might forget to keep this up-to-date when we create new state nodes. are there ones we want to ignore that are noisy? maybe let's just track all, might be easier to maintain. i'd suggest maybe just ignoring `idle`?

Reply:
There are a bunch that we probably don't want. Pretty much all `Idle`, `Pointing*`, probably others.
![CleanShot 2024-04-17 at 13 48 42](https://github.com/tldraw/tldraw/assets/2523721/0c6fd4af-20ec-43f5-bdbe-b87a79955fb4)

We could go with a blacklist approach though. Then we don't miss any new ones and if the new ones are kinda pointless to track they will annoy someone and they will add it to the blacklist?